### DISCUSSION TITLE
Nepali Translation

### DISCUSSION THREAD

**Discussion by sumn2u** [sumn2u/learn-javascript#9] [Show and tell]
I would like to extend my sincere appreciation and gratitude to all the contributors who played a crucial role in translating the content of the "Learn JavaScript" book into Nepali. Your efforts and dedication have made a significant difference in making this valuable resource accessible to a wider audience. Thank you for your outstanding contribution and commitment to making the world of programming more inclusive. 👏

Contributors
@Anuj-Khadka 
@RupeshBhandari 
@sahilxkhadka

**Comment by Anuj-Khadka** [comment]
Thank you so much for the opportunity. Have been a little busy recently. I will still try to contribute as much as I can after this  month.

**Reply by myselfprincee** [reply]
It is the best time. you should contribute now.
